configure.ac

```diff
@@ -904,6 +929,16 @@ libcoap configuration summary:
       libcoap API version      : "$LIBCOAP_API_VERSION"
       libcoap DTLS lib extn    : "$LIBCOAP_DTLS_LIB_EXTENSION_NAME"
       host system              : "$host"]);
+if test "x$enable_server_mode" = "xyes"; then
+    AC_MSG_RESULT([      build wih Server support : "yes"])
```

Comment:
Here and for the following three messages: "wih" → "with"

And I suggest lower case for "client" and "server".

Reply:
Corrected